src/helper.py

```diff
@@ -27,9 +27,9 @@ def arguments_dictionary(lat, long, city, args):
         "show_period": 1,
         "show_city": 1,
         "show_date": 1,
-        "show_air_temp": 0,
-        "show_wind_speed": 0,
-        "show_wind_direction": 0,
+        "show_air_temp": 1,
+        "show_wind_speed": 1,
+        "show_wind_direction": 1,
```

Comment:
These should all be set to 0, and only show if the user wants them to (via a command line argument, like `show_wind_speed`